pubspec.yaml

```diff
@@ -15,7 +15,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 0.2.3+6
+version: 0.2.5+1
```

Comment:
The `+6` part is used by the Play store to distinguish unique versions apart from the SemVer (e.g. `0.2.5`). So it is vital that it is incremented for each release. Is it a problem for iOS to leave it at 6 or higher?

Reply:
For iOS everything after the `+` is treated as the build number. Problem here is the build number doesn't count towards the unique identifier (on the App Store). But each upload to App Store Connect requires a unique identifier. In conclusion for future iOS releases we need to stick to the `0.2.5` for this MR. If you even need to increment the build number for minor or major releases we can do so.